ulauncher/app.py

```diff
@@ -171,5 +171,5 @@ def toggle_hold(self, value: bool) -> None:
         self.hold() if value else self.release()
 
     @events.on
-    def quit(self) -> None:
+    def quit(self) -> None:  # pyrefly: ignore[bad-override]
```

Comment:
I don't actually understand this error, but it works now, so I ignored it:

`UlauncherApp.quit` has type `BoundMethod[UlauncherApp, (...) -> Any]`, which is not assignable to `BoundMethod[UlauncherApp, (self: UlauncherApp) -> None]`, the type of `Application.quit`